collect_app/src/main/java/org/odk/collect/android/activities/viewmodels/SplashScreenViewModel.kt

```diff
@@ -27,15 +31,23 @@ class SplashScreenViewModel(
     val doesLogoFileExist
         get() = splashScreenLogoFile.exists()
 
-    fun shouldFirstLaunchDialogBeDisplayed(): Boolean {
-        val isFirstLaunch = !metaSettings.contains(MetaKeys.KEY_FIRST_LAUNCH)
-        metaSettings.save(MetaKeys.KEY_FIRST_LAUNCH, false)
-        return isFirstLaunch
+    val isFirstLaunch
+        get() = appStateProvider.isFreshInstall()
+
+    fun importExistingProjectIfNeeded() {
```

Comment:
Would it make sense to have this happen in the `ApplicaitonInitializer` instead?

Reply:
Yeah, I agree. Fixed.